Source/WebCore/dom/FullscreenManager.h

```diff
@@ -47,7 +47,6 @@ class FullscreenManager final : public CanMakeWeakPtr<FullscreenManager> {
 
     Document& document() { return m_document; }
     const Document& document() const { return m_document; }
-    Document& topDocument() const { return m_document.topDocument(); }
```

Comment:
Given how it is used, we may want to make this getter private instead and have it return `document()` when `m_topDocument` is null. It would simplify the call sites.